images/custom/Containerfile

```diff
@@ -64,7 +64,7 @@ RUN useradd -ms /bin/bash frappe \
     && chown -R frappe:frappe /var/log/nginx \
     && chown -R frappe:frappe /var/lib/nginx \
     && chown -R frappe:frappe /run/nginx.pid \
-    && chmod +x /usr/local/bin/nginx-entrypoint.sh
+    && chmod 755 /usr/local/bin/nginx-entrypoint.sh
 
```

Comment:
Please also add "&& chmod 644 /templates/nginx/frappe.conf.template" after line 67

Reply:
Why 644 for this file?
What problem are you facing?

Reply:
I think there is no problem as long as the template available for user frappe to read and generate config from.

The comment on the PR states the permissions that are in frappe/erpnext image and the `chmod 644` will make sure the permission is set to same.
 
- Original image doesn't do this acrobatics. It is built with linux (gh actions ubuntu runners). Use linux runners or linux machines to avoid additional workarounds. May be there are hidden ones yet to come?